packages/api/src/hooks/useMT5AccountsList.ts

```diff
@@ -44,7 +44,7 @@ const useMT5AccountsList = () => {
             /** The id of the account */
             loginid: account.login,
             /** The platform of the account */
-            platform: 'mt5',
+            platform: 'mt5' as const,
```

Comment:
The `useMT5AccountsList` platform attribute returns it as a `string`, not `mt5`:
<img width="778" alt="Screenshot 2023-10-26 at 6 36 05 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/df23a87f-80d3-4bb4-b2e5-e7174e1563a4">
This affects all the typing for `TPlatforms.All` to be casted into `string` instead of `mt5 | xtrade | ctrader`:
<img width="392" alt="Screenshot 2023-10-26 at 6 36 41 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/b46b3c17-60a2-4822-8313-6f6f5e3a81ff">
Thus by narrowing it down, we ensure `TPlatforms.All` has types `mt5 | ctrader | xtrade` instead of being widened into `string`
<img width="524" alt="Screenshot 2023-10-26 at 6 37 30 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/8cd1f9d7-9a8d-4fdd-b6d6-a6dda15bd529">